xml/System.Globalization/RegionInfo.xml

```diff
@@ -181,6 +177,7 @@
         </remarks>
         <exception cref="T:System.ArgumentException">
           <paramref name="culture" /> specifies either an invariant, custom, or neutral culture.</exception>
+        <exception cref="T:System.Globalization.CultureNotFoundException">.NET 6+ only: Globalization-invariant mode is enabled in the environment, and <paramref name="culture" /> is not the invariant culture.</exception>
```

Comment:
@tarekgh Is this the correct exception for the RegionInfo constructor?

Reply:
In the case of the `RegionInfo`, will get the exception like

```
System.ArgumentException: Culture name 'en-US' is not supported. (Parameter 'name')
```

but that is because Globalization Invariant mode is enabled.